docker-compose.yml

```
@@ -51,10 +83,10 @@ services:
     volumes:
       - ./:/app
     command: >
-      bash -c "sleep 10 && alembic upgrade head && python /app/scripts/ingest_joplin.py"
+      bash -c "sleep 10 && cd stac_fastapi/sqlalchemy && alembic upgrade head && python scripts/ingest_joplin.py"
```

Migration is broken with this change, fixed by:

```suggestion
      bash -c "sleep 10 && cd stac_fastapi/sqlalchemy && alembic upgrade head && cd ../.. && python scripts/ingest_joplin.py"
```